core/capabilities/syncer.go

```diff
@@ -66,10 +79,22 @@ func (s *registrySyncer) Start(ctx context.Context) error {
 		"12D3KooW9zYWQv3STmDeNDidyzxsJSTxoCTLicafgfeEz9nhwhC4",
 	}
 	triggerDONPeers := []string{
-		"12D3KooWJrthXtnPHw7xyHFAxo6NxifYTvc8igKYaA6wRRRqtsMb",
-		"12D3KooWFQekP9sGex4XhqEJav5EScjTpDVtDqJFg1JvrePBCEGJ",
-		"12D3KooWFLEq4hYtdyKWwe47dXGEbSiHMZhmr5xLSJNhpfiEz8NF",
-		"12D3KooWN2hztiXNNS1jMQTTvvPRYcarK1C7T3Mdqk4x4gwyo5WS",
+		"12D3KooWBaiTbbRwwt2fbNifiL7Ew9tn3vds9AJE3Nf3eaVBX36m",
+		"12D3KooWS7JSY9fzSfWgbCE1S3W2LNY6ZVpRuun74moVBkKj6utE",
+		"12D3KooWMMTDXcWhpVnwrdAer1jnVARTmnr3RyT3v7Djg8ZuoBh9",
+		"12D3KooWGzVXsKxXsF4zLgxSDM8Gzx1ywq2pZef4PrHMKuVg4K3P",
+		"12D3KooWSyjmmzjVtCzwN7bXzZQFmWiJRuVcKBerNjVgL7HdLJBW",
+		"12D3KooWLGz9gzhrNsvyM6XnXS3JRkZoQdEzuAvysovnSChNK5ZK",
+		"12D3KooWAvZnvknFAfSiUYjATyhzEJLTeKvAzpcLELHi4ogM3GET",
+	}
+	triggerDONSigners := []string{
+		"0x9CcE7293a4Cc2621b61193135A95928735e4795F",
+		"0x3c775F20bCB2108C1A818741Ce332Bb5fe0dB925",
+		"0x50314239e2CF05555ceeD53E7F47eB2A8Eab0dbB",
+		"0xd76A4f98898c3b9A72b244476d7337b50D54BCd8",
+		"0x656A873f6895b8a03Fb112dE927d43FA54B2c92A",
+		"0x5d1e87d87bF2e0cD4Ea64F381a2dbF45e5f0a553",
+		"0x91d9b0062265514f012Eb8fABA59372fD9520f56",
```

Comment:
Heads up, that this will change to be `bytes32` onchain (https://github.com/smartcontractkit/chainlink/pull/13183/files#diff-d94c1796b3544ef8780af7e16b5361c8fb9e4d63f492faeda6a3362a8363f51bR45-R51). We can do decoding once we Syncer is up, but not sure the best place to do that, thread: https://chainlink-core.slack.com/archives/C064418MJHE/p1716308390423929

Reply:
That's fine. As long as the Syncer passes correct binary addresses to all services, we're good.